src/main.cpp

```diff
@@ -498,23 +509,33 @@ int main ( int argc, char** argv )
 
     if ( !strServerPublicIP.isEmpty() )
     {
-        QHostAddress InetAddr;
-        if ( !InetAddr.setAddress ( strServerPublicIP ) )
+        if ( bIsClient || strCentralServer.isEmpty() )
         {
-            qWarning() << "Server Public IP is invalid. Only plain IP addresses are supported.";
+            qWarning() << "Server Public IP will only take effect when registering a server with a directory server.";
         }
-        if ( strCentralServer.isEmpty() || bIsClient )
+        else
         {
-            qWarning() << "Server Public IP will only take effect when registering a server with a directory server.";
+            QHostAddress InetAddr;
+            if ( !InetAddr.setAddress ( strServerPublicIP ) )
+            {
+                qWarning() << "Server Public IP is invalid. Only plain IP addresses are supported.";
+            }
         }
     }
 
     if ( !strServerBindIP.isEmpty() )
     {
-        QHostAddress InetAddr;
-        if ( !InetAddr.setAddress ( strServerBindIP ) )
+        if ( bIsClient )
+        {
+            qWarning() << "Server Bind IP will only take effect when running a server.";
+        }
+        else
         {
-            qWarning() << "Server Bind IP is invalid. Only plain IP addresses are supported.";
+            QHostAddress InetAddr;
+            if ( !InetAddr.setAddress ( strServerBindIP ) )
+            {
+                qWarning() << "Server Bind IP is invalid. Only plain IP addresses are supported.";
```

Comment:
Only plain IP-addresses are supported. --> Please check that the IP you entered is valid.

Reply:
That removes information.

Reply:
It's meant to imply you entered a hostname, rather than an IP address.  I don't think it cares much whether the IP is _valid_, either.